index.md

```diff
@@ -1,131 +1,198 @@
+stimulate the economy with the theory that money would eventually reach the people.
 
-Most of the time that never happened and during the Coronavirus pandemic, we are seeing airlines pay massive dividends at the same time as they are being bailed out. If we bail out corporations, we’re only going to increase the divide between the rich and poor, leading to greater desperation and poverty which will drive up crime.  
+Most of the time that never happened and during the Coronavirus pandemic, we are seeing airlines pay massive 
+dividends at the same time as they are being bailed out. If we bail out corporations, we’re only going to 
+increase the divide between the rich and poor, leading to greater desperation and poverty which will drive 
+up crime.  
 
-Instead of bailing out corporations, who will not look out for citizens, governments should bail out the citizens themselves. Citizens will have the power to restart their life and will choose where to spend the government support money. This in turn will benefit the companies that the citizens need the most and the citizens of the country will decide which companies should survive and which ones should be left behind. 
+Instead of bailing out corporations, who will not look out for citizens, governments should bail out the 
+citizens themselves. Citizens will have the power to restart their life and will choose where to spend 
+the government support money. This in turn will benefit the companies that the citizens need the most and 
+the citizens of the country will decide which companies should survive and which ones should be left behind. 
 
 
 Focus on climate change and our environment
 -------------------------------------------
 
-The coronavirus pandemic has also shown how bad pollution is in this world. The most polluted cities suddenly have blue skies now that drastically fewer cars and planes are polluting our environment. It also showed that people who chronically live with bad air are much more likely to be negatively impacted by the coronavirus.
+The coronavirus pandemic has also shown how bad pollution is in this world. The most polluted cities suddenly 
+have blue skies now that drastically fewer cars and planes are polluting our environment. It also showed that 
+people who chronically live with bad air are much more likely to be negatively impacted by the coronavirus.
 
-We can take the lessons learned during the pandemic and reduce our impact on our planet? How can we commute less? Can we have more online meetings and travel less to business meetings? What sustainable technologies can we invest in to displace the polluting technologies?
+We can take the lessons learned during the pandemic and reduce our impact on our planet? How can we commute less? 
+Can we have more online meetings and travel less to business meetings? What sustainable technologies can we 
+invest in to displace the polluting technologies?
```

Comment:
Instead of asking questions, what about being more affirmative? (damn, that's a question...)

Something like:

We can take the lessons learned during the pandemic and reduce our impact on our planet.
Having more online meetings, reducing our travels to needed ones, and investing in sustainable technologies ...